checkov/terraform/checks/resource/oci/IAMPasswordLength.py

```diff
@@ -16,7 +16,7 @@ def scan_resource_conf(self, conf):
             rules = conf.get("password_policy")[0]
             if 'minimum_password_length' in rules:
                 passwordlength = rules.get("minimum_password_length")
-                if passwordlength[0] < 14:
+                if isinstance(passwordlength[0], int) and passwordlength[0] < 14:
```

Comment:
@YaaraVerner Please use `force_int` method 🙏

Reply:
the use-case that an exception was thrown is when the attribute is assign with variable, force_int won't work for this

Reply:
Resolving and writing it again 😓